### DISCUSSION TITLE
How can I return robots.txt ?

### DISCUSSION THREAD

**Discussion by andriiradkevych** [freshframework/fresh#2201] [Q&A] [answered]
I need to make `robots.tsx` on routing . I have created file `routes/robots.tsx.ts` file , but don't know what to do next , can I do it ? Just return plain text

```
User-agent: *
Disallow: /
```

**Comment by marvinhagemeister** [comment, marked as answer]
The `robots.txt` file is a text file and thus not a JavaScript file that needs to be executed. All static files like images, videos or other types belong in the `static/` directory. Everything in the `static` directory is accessible directly. So if you put a file called `robots.txt` in there, it will be accessible via `http://<my-domain>/robots.txt`.

You can find more information about static files and Fresh on the [Static Files](https://fresh.deno.dev/docs/concepts/static-files) page in our documentation.

**Reply by andriiradkevych** [reply]
It doesn't . I've made it , and it return me `404` page

**Reply by marvinhagemeister** [reply]
I just tried it with a newly bootstrapped Fresh project and it works as expected. Based on that I'm assuming that there must be something different with your setup. Can you share the exact folder where you put it? It should be located at `<project>/static/robots.txt`.

**Reply by andriiradkevych** [reply]
yeah , sorry , my mistake , leaved `robots.txt.ts` in `routes` folder, thank you for your help
